package.json

```diff
@@ -0,0 +1,67 @@
+{
+	"name": "algobuilder-msig",
+	"description": "Multisignature Sign UI",
+	"homepage": "https://algobuilder.dev",
+	"license": "mpl-2.0",
+	"engines": {
+		"node": ">=14.11.0",
+		"yarn": ">=3.1.0"
+	},
+	"scripts": {
+		"dev": "vue-cli-service serve",
+		"server": "nodemon backend/server.ts",
```

Comment:
You don't need this server script.

Reply:
The script "dev" we should keep to run the server too!

Reply:
So I will remove `"server": "nodemon backend/server.ts"

Reply:
Yes